Apps/WebClient/src/ClientApp/src/components/credential/credentialList.vue

```diff
@@ -96,6 +105,10 @@ export default class CredentialList extends Vue {
         return credential?.status === CredentialStatus.Added;
     }
 
+    private isMenuDisabled(credential: WalletCredential): boolean {
+        return credential.status !== CredentialStatus.Created || this.isLoading;
```

Comment:
The menu should be enabled when the credential has either a Created status or Added status.  It's being disabled currently for the Added status because of this check.

Reply:
We should chat about this since its related to the convo we had in the morning